server/routes/createTodos.js

```diff
@@ -3,6 +3,16 @@ const { connect } = require('hyper-connect')
 const hyper = connect(process.env.HYPER)
 
 module.exports = async function createTodos (req, res) {
-  const result = await hyper.data.add(req.body)
-  res.json(result)
+  console.log(req.body)
+  try {
+    if (!req.body.task) {
+      res.status(400).json({ message: 'Add a new task' })
+    } else {
+      const newTodo = { task: req.body.task, completed: req.body.completed }
```

Comment:
eventually, you will want to validate the shape of the incoming `body` to ensure it has a `task` and `completed` _and_ ensure they are the correct type ie. `string` or `boolean`. This is commonly called "input sanitizing"

A library we use at hyper is called [zod](https://www.npmjs.com/package/zod) . There are others out there like `joi` and `yup`.

Reply:
Thanks